code/src/index.css

```
@@ -11,3 +14,163 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+h1, h2, p, h4, .Submit {
+  font-family: "Trispace";
+}
+
+h1 {
+  font-size: 24px;
+  text-align: center;
+  display: block;
+  margin: 30px auto 20px;
+  font-weight: 550;
+  
+}
+
+h2 {
+  font-size: 16px;
+  font-weight: 600;
+  margin: 3.5%;
+}
+
+h4 {
+  font-size: 16px;
+  font-weight: 400;
+  margin: 0px 0 20px 0;
+  width: 100%;
+  display: flex;
+}
+
+p {
+color: grey;
+font-size: 14px;
+}
+
+.PostThought {
+  width: auto;
+  margin: 0 30% 0 30%;
+  height:  200px;
```

I noticed that the position of the button was bugging out a bit on larger screens and when resizing the window. Setting height to "fit-content" seemed to fix it, at least in my dev tools 🙂

```suggestion
  height:  fit-content;
```